app/embedders/twitter_embedder.rb

```diff
@@ -0,0 +1,43 @@
+class TwitterEmbedder < OpenGraphEmbedder
+  def twitter(key)
+    page.at_css("meta[name='twitter:#{key}']")&.[]('content')
+  end
+
+  def player_info
+    {
+      url: twitter('player'),
+      width: twitter('player:width'),
+      height: twitter('player:height')
+    }.compact
+  end
+
+  def card
+    twitter :card
+  end
+
+  def site
+    twitter :site
+  end
+
+  def creator
+    twitter :creator
+  end
+
+  def twitter_info
+    {
+      card: card,
+      site: site,
+      creator: creator,
+      twitter: twitter,
+      player_info: player_info
```

Comment:
We should try to keep this structure uniform across all embed systems so that the client doesn't need to know where the data came from.  When I wrote the OpenGraphEmbedder, I didn't change much because it was already pretty much a reasonable structure, but we should try to build an abstracted JSON structure for embeds